third-party/crowdsec/pom.xml

```diff
@@ -40,6 +40,19 @@
             <type>test-jar</type>
             <scope>test</scope>
         </dependency>
```

Comment:
Please use `scope provided` for dependencies shiped as part of the james container in order to reduce the size of your extension.

Reply:
Keep test on that one!